charts/argo-cd/templates/_helpers.tpl

```diff
@@ -183,24 +185,25 @@ Merge Argo Configuration with Preset Configuration
 
 {{/*
 Argo Params Default Configuration Presets
+NOTE: Configuration keys must be stored as dict because YAML treats dot as separator
 */}}
 {{- define "argo-cd.config.params.presets" -}}
-repo.server: "{{ include "argo-cd.repoServer.fullname" . }}:{{ .Values.repoServer.service.port }}"
-server.repo.server.strict.tls: {{ .Values.repoServer.certificateSecret.enabled | toString }}
-{{- with include "argo-cd.redis.server" . }}
-redis.server: {{ . | quote }}
-{{- end }}
-{{- if .Values.dex.enabled }}
-server.dex.server: {{ include "argo-cd.dex.server" . | quote }}
-server.dex.server.strict.tls: {{ .Values.dex.certificateSecret.enabled | toString }}
-{{- end }}
-{{- range $component := tuple "applicationsetcontroller" "controller" "server" "reposerver" }}
-{{ $component }}.log.format: {{ $.Values.global.logging.format | quote }}
-{{ $component }}.log.level: {{ $.Values.global.logging.level | quote }}
-{{- end }}
-{{- if .Values.applicationSet.enabled }}
-applicationsetcontroller.enable.leader.election: {{ gt (.Values.applicationSet.replicaCount | int64) 1 }}
-{{- end }}
+{{- $presets := dict -}}
+{{- $_ := set $presets "repo.server" (printf "%s:%s" (include "argo-cd.repoServer.fullname" .) (.Values.repoServer.service.port | toString)) -}}
+{{- $_ := set $presets "server.repo.server.strict.tls" (.Values.repoServer.certificateSecret.enabled | toString ) -}}
+{{- $_ := set $presets "redis.server" (include "argo-cd.redis.server" .) -}}
+{{- if .Values.dex.enabled -}}
+{{- $_ := set $presets "server.dex.server" (include "argo-cd.dex.server" .) -}}
+{{- $_ := set $presets "server.dex.server.strict.tls" .Values.dex.certificateSecret.enabled -}}
+{{- end -}}
+{{- range $component := tuple "applicationsetcontroller" "controller" "server" "reposerver" -}}
+{{- $_ := set $presets (printf "%s.log.format" $component) $.Values.global.logging.format -}}
+{{- $_ := set $presets (printf "%s.log.format" $component) $.Values.global.logging.format -}}
```

Comment:
@pdrastil I think you've introduced a regression by removing possibility to set `logging.level` globally. Lines 200 and 201 are the same but `201` should look like

```yaml
{{- $_ := set $presets (printf "%s.log.level" $component) $.Values.global.logging.level -}}
```

Reply:
@milpog - You are right, thanks for letting me know. Fix is on the way.

Reply:
Thank you 🤗